db/queries/users.sql

```diff
@@ -0,0 +1,17 @@
+-- name: SelectUsers :many
+select * from users;
+
+-- name: AddUser :exec
+insert into users
+(email, username, pass_hash, pass_salt, role)
+values
+($1,$2,$3, $4, $5);
```

Comment:
It would be nice if the insert command also returned the added User.
```suggestion
($1, $2, $3, $4, $5)
RETURNING *;
```